public/index.html

```diff
@@ -19,7 +19,10 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>CarbonDoomsDay</title>
+    <title>Carbon Doomsday - Climate change chart & carbon dioxide levels</title>
```

Comment:
indeed! https://stackoverflow.com/questions/3493405/do-i-really-need-to-encode-as-amp